ebos/eclproblem.hh

```diff
@@ -2571,6 +2573,12 @@ private:
         if (FluidSystem::enableVaporizedOil() && !has_rv)
             throw std::runtime_error("The ECL input file requires the RV keyword to be present if"
                                      " vaporized oil is enabled");
+        if (enableBrine && !has_salt)
+            throw std::runtime_error("The ECL input file requires the SALT keyword to be present if"
+                                     " brine is enabled");
```

Comment:
Isn't there a change of semantics here?

It seems that now an error is thrown if you simulate brine and did not explicitly specify SALT (`has_salt==false`). I assume that previously the default would have been used.

Reply:
The SALT keyword is only required if brine is enabled _and_ the model is initialized  explicitly. The modified part of the code deals with the explicit initialization.  I have improved the error message.